src/Clear.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace SilverStripe\GraphQLDevTools;
+
+use Psr\Log\LoggerInterface;
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\GraphQL\Schema\Storage\CodeGenerationStore;
+use Symfony\Component\Filesystem\Filesystem;
+
+class Clear extends Controller
+{
+    private static $url_handlers = [
+        '' => 'clear',
+    ];
+
+    private static $allowed_actions = [
+        'clear',
+    ];
+
+    public function clear(HTTPRequest $request): void
+    {
+
+        $logger = Injector::inst()->get(LoggerInterface::class . '.graphql-build');
+        $dirName = CodeGenerationStore::config()->get('dirName');
+        $expectedPath = BASE_PATH . DIRECTORY_SEPARATOR . $dirName;
+        $fs = new Filesystem();
+
+        $logger->info('Clearing GraphQL code generation directory');
+        if ($fs->exists($expectedPath)) {
+            $logger->info('Directory has been found');
+            $fs->remove($expectedPath);
```

Comment:
Should it remove the whole directory? I can think of situations where risk averse sysadmins may create the directory themselves and grant write access only to that directory, which means `dev/graphql/build` won't be able to re-create the directory after this.

Reply:
Changed the logic to remove the children instead.